extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -115,28 +115,81 @@ function render_player( $player_data, $attributes ) {
 		$player_data
 	);
 
+	// Set CSS classes for `secondary` and `background` colors.
+	$secondary_color         = isset( $attributes['secondaryColor'] ) ? $attributes['secondaryColor'] : null;
+	$custom_secondary_color  = isset( $attributes['customSecondaryColor'] ) ? $attributes['customSecondaryColor'] : null;
+	$background_color        = isset( $attributes['backgroundColor'] ) ? $attributes['backgroundColor'] : null;
+	$custom_background_color = isset( $attributes['customBackgroundColor'] ) ? $attributes['customBackgroundColor'] : null;
+
+	$secondary_color_class  = get_color_class_name( 'color', $secondary_color );
+	$background_color_class = get_color_class_name( 'background-color', $background_color );
+
+	$secondary_classes_name = '';
+	$secondary_inline_style = '';
+	if ( isset( $secondary_color_class ) || isset( $custom_secondary_color ) ) {
+		$secondary_classes_name .= ' has-secondary';
+		if ( isset( $secondary_color_class ) ) {
+			$secondary_classes_name .= $secondary_color_class;
+		} elseif ( isset( $custom_secondary_color ) ) {
+			$secondary_inline_style .= "color: $custom_secondary_color;";
+		}
+	}
+
+	$background_classes_name = '';
+	$background_inline_style = '';
+	if ( isset( $background_color_class ) || isset( $custom_background_color ) ) {
+		$background_classes_name .= ' has-background';
+		if ( isset( $background_color_class ) ) {
+			$background_classes_name .= $background_color_class;
+		} elseif ( isset( $custom_background_color ) ) {
+			$background_inline_style .= "background-color: $custom_background_color;";
+		}
+	}
```

Comment:
It would probably be easier to read if all of the above was in a separate function

Reply:
Not sure. We've worked previously in the Navigation menu and honestly, I'm don't think that splitting the code out in helper functions will improve the readability.